Client/Frontend/Home/HomepageHeroImageViewModel.swift

```diff
@@ -0,0 +1,22 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import SiteImageView
+
+struct HomepageHeroImageViewModel: HeroImageViewModel {
+    let urlStringRequest: String
+    let type: SiteImageView.SiteImageType = .heroImage
+    let generalCornerRadius: CGFloat = HomepageViewModel.UX.generalCornerRadius
+    let faviconCornerRadius: CGFloat = HomepageViewModel.UX.generalCornerRadius
+    let faviconBorderWidth: CGFloat = HomepageViewModel.UX.generalBorderWidth
+    let heroImageSize: CGSize
+    let fallbackFaviconSize: CGSize = HomepageViewModel.UX.fallbackFaviconSize
+
+    init(urlStringRequest: String,
```

Comment:
small nit: you could remove the init structs provide default initializer

Reply:
Oh right, I got confused since I needed to create inits for structs in SiteImageView package (since by default the init is internal and I needed it public since it's a package). Thanks!